src/Raven.Studio/typescript/viewmodels/manage/certificates.ts

```diff
@@ -280,6 +281,21 @@ class certificates extends viewModelBase {
         eventsCollector.default.reportEvent("certificates", "generate");
         this.model(certificateModel.generate());
     }
+
+    enterReGenerateCertificateMode(itemToReGenerate: unifiedCertificateDefinition) {
+        eventsCollector.default.reportEvent("certificates", "re-generate");
+        this.model(certificateModel.generate());
+
+        this.model().name(itemToReGenerate.Name);
```

Comment:
will that override old cert or 2 with same name are allowed?

Reply:
2 with the same name are allowed, as long as the generated thumbprint is different